README.md

```diff
@@ -138,16 +258,31 @@ Always close the tracer before exiting your application to flush all buffered sp
 tracer.close();
 ```
 
+## Span Logs 
+
+You can instrument your application to emit logs or events with spans, and examine them from the [Tracing UI](https://docs.wavefront.com/tracing_ui_overview.html#drill-down-into-spans-and-view-metrics-and-span-logs).
```

Comment:
...and examine them from the [Tracing UI]...
...and examine them from the [Wavefront Tracing UI]

(Good to be specific here, I think.)

Reply:
Good point! Updated :)